week-2/implement/vat.js

```diff
@@ -8,3 +8,16 @@
 // Given a number,
 // When I call this function with a number
 // it returns the new price with VAT added on
+
+function withVat(message, price) {
+  const vatMessage = `The price of your ${message} is £${price * 1.2}`;
+  return vatMessage;
```

Comment:
This is good. Here, I would probably just return the new (VAT included) price rather than the message - but...it depends on the exact requirement. 

I'd give you Pro-points for making the VAT rate a parameter. Double pro-points for making the VAT rate vary depending on the type of good.